recipes/inspiral-range/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "inspiral_range" %}
+{% set version = "0.3.0" %}
+
+
+package:
+  name: {{ name|lower|replace('_', '-') }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5cd901bb6c3a328dc048dc529a5499a0cdec9f4ed102e9835a80807cb330af9f
+
+build:
+  entry_points:
+    - inspiral-range = inspiral_range.__main__:main
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - astropy
+    - python
+    - scipy
+
+test:
+  requires:
+    - python-lal  # [not win]
```

Comment:
If it is required for the tests, why isn't it a `run` dependency?

Reply:
`python-lal` is an optional dependency, and having provides a more robust test, but it isn't available on windows, so it shouldn't be a full dependency.

Reply:
python-lal is not actually required for the tests either.

But python-lal should be preferred as a runtime/test dependency over python-astropy, if it's possible to express that somehow...